softaculous_plugin.php

```
@@ -32,19 +57,106 @@ public function getEvents()
         ];
     }
 
-    public function softAutoInstall($event)
+    /**
+     * Retrieves cron tasks available to this plugin along with their default values
+     *
+     * @return array A list of cron tasks
+     */
+    private function getCronTasks()
+    {
+        return [
+            // Cron task to reattempt script installation
+            [
+                'key' => 'reattempt_installation',
```

I would just call this "script_installation" since that is its purpose, rather than assuming it must have failed previously.